hashtagmap-instagram-crawler/src/main/java/com/songpapeople/hashtagmap/util/PlaceNameParser.java

```diff
@@ -1,12 +1,14 @@
 package com.songpapeople.hashtagmap.util;
 
-public class PlaceNameParser {
-    private static final int FIRST_INDEX = 0;
+import java.util.Optional;
 
+public class PlaceNameParser {
     public static String parsePlaceName(String placeName) {
         String parsedPlaceName = placeName.replaceAll(" ", "");
-        if (parsedPlaceName.endsWith("점")) {
-            return parsedPlaceName.substring(FIRST_INDEX, parsedPlaceName.length() - 1);
+        Optional<PlaceNameType> optionalPlaceNameType = PlaceNameType.find(parsedPlaceName);
+        if (optionalPlaceNameType.isPresent()) {
+            PlaceNameType placeNameType = optionalPlaceNameType.get();
+            return placeNameType.parsePlaceName(parsedPlaceName);
         }
         return parsedPlaceName;
```

Comment:
```suggestion
        Optional<PlaceNameType> optionalPlaceNameType = PlaceNameType.find(parsedPlaceName);
        return optionalPlaceNameType
                .map(placeNameType -> placeNameType.parsePlaceName(parsedPlaceName))
                .orElseGet(()->parsedPlaceName);
```

isPresent 말고 이렇게 하면 더 좋을것 같아요!

Reply:
굿 :)